### PR TITLE
Use movement action constant values over enum

### PR DESCRIPTION
**Pull request by cbt6**
## Description
Also added movement actions for Acro Bike for completion's sake.

## **Discord contact info**
cbt#8922

### CODE REVIEW

File: asm/macros/movement.inc [hunk 1]
```diff
@@ -1,131 +1,166 @@
-	.macro create_movement_action name
-	enum _\name
+	.macro create_movement_action name, value
 	.macro \name
```
Comment (huderlem) [inline asm/macros/movement.inc:2]: These should probably just be constants instead of macros, I think?  The macros simply output a single byte.

Comment (GriffinRichards) [reply asm/macros/movement.inc:2]: The macros are simplified names for the constants (plus the byte directive). Movement scripts would be unwieldy with the C defines

File: asm/macros/movement.inc [hunk 2]
```diff
@@ -1,131 +1,166 @@
-	.macro create_movement_action name
-	enum _\name
+	.macro create_movement_action name, value
```
Comment (GriffinRichards) [inline asm/macros/movement.inc:1]: Looks like the old macro was created before we started adding argument requirements.
Should be `.macro create_movement_action name:req, value:req`

File: asm/macros/movement.inc (hunk 1 above)
Comment (huderlem) [reply asm/macros/movement.inc:2]: True, I forgot about that.

**Pull request merged**